menas/src/main/scala/za/co/absa/enceladus/menas/exceptions/BadRequestException.scala

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2018-2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.menas.exceptions
+
+case class BadRequestException(message: String, cause: Throwable = None.orNull) extends Exception(message, cause)
```

Comment:
Avoiding the `null` warning? 😉 
While smart, doesn't it more obscure the code, then help?